docs/content/SelectPanel.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: SelectPanel
+status: Alpha
+---
+
+A `SelectPanel` provides an anchor that will open an overlay with a list of selectable items, and a text input to filter the selectable items
+
+## Example
+
+## Component props
```

Comment:
1. Would you mind opening an issue tracking either a. “Promote `SelectPanel` to “beta” status` or b. “Provide examples in the `SelectPanel` docs”?
2. What do you think about incorporating some of the prop documentation from https://github.com/primer/components/issues/1071 and https://github.com/primer/components/issues/1070?